contracts/BondingEvent.sol

```diff
@@ -338,14 +342,17 @@ contract BondingEvent is AccessControl {
             int24 upperTick
         ) = getOtherAmount(_amountSEuro);
 
+
+        // Gives a 0.01 slippage tolerance to sEURO
+        // Due to the possible difference in accuracy between sEURO (18dec) and another token (possible 6dec)
```

Comment:
this pr now adds a 0.01% slippage tolerance for seuro.
this is because seuro is much more accurate than a 6 dec token (which we will probably use). therefore, althought `getOtherAmount` calculates the nearest ratio amount for a token to the seuro input, there is unlikely to be an amount that is EXACTLY proportionate to the very accurate seuro input amount.

this means that we have to be a bit more lenient with the seuro amounts (we're talking about <0.000001 sEURO per transaction